docs/antora.yml

```diff
@@ -1,5 +1,5 @@
 name: quarkus-micrometer-registry
-title: Quarkus Micrometer Registry extensions
+title: Micrometer Registry
```

Comment:
I think the title is ok... This project is a multi module project with many Micrometer registries to use in Quarkus.

Reply:
I just wanted to ask if this repo isn't a set of Quarkus extensions for optional Micrometer registries?

Reply:
Yes it is @jonatan-ivanov